src/core/fair_queue.cc

```diff
@@ -243,28 +242,24 @@ void fair_queue::notify_request_cancelled(fair_queue_entry& ent) noexcept {
     ent._capacity = 0;
 }
 
-fair_queue::clock_type::time_point fair_queue::next_pending_aio() const noexcept {
-    return _throttle.next_pending();
-}
-
 void fair_queue::dispatch_requests() {
     capacity_t dispatched = 0;
     boost::container::small_vector<priority_class_ptr, 2> preempt;
 
-    while (!_handles.empty() && (dispatched < _throttle.per_tick_grab_threshold())) {
```

Comment:
Minor: `dispatched` variable becomes unused after the removal. The compiler on CI complains about it:

```
fair_queue.cc:238:16: error: variable 'dispatched' set but not used [-Werror,-Wunused-but-set-variable]
  238 |     capacity_t dispatched = 0;
```

Reply:
I am not entirely sure why the compiler raises error -- below there is `dispatched += req_cap;`

Reply:
Because this variable becomes write-only, which effectively means unused. This place is obsoleted with #2294 anyway :(